pilot/pkg/bootstrap/namespacecontroller.go

```diff
@@ -61,6 +65,32 @@ func NewNamespaceController(ca *ca.IstioCA, core corev1.CoreV1Interface) (*Names
 		core: core,
 	}
 
+	configMapLw := listwatch.MultiNamespaceListerWatcher([]string{metav1.NamespaceAll}, func(namespace string) cache.ListerWatcher {
+		return &cache.ListWatch{
```

Comment:
Does this need to be listwatch.MultiNamespaceListerWatcher? Or is this future proofing for if/when istiod which watch subsets of namespacs?

Reply:
What is the alternative? I couldn't find another way to do the label selection as well

Reply:
`cache.NewFilteredListWatchFromClient` 

https://github.com/kubernetes/client-go/blob/8a503c805acfead184993607cd87332414387a68/tools/cache/listwatch.go#L80

Reply:
I couldn't get this to work, I think it doesn't work for namespaces potentionally?

Reply:
Huh. A quick glance at the code shows that an empty namespace is not included in the URL. You had something like the following?

```golang
namespaceLW := cache.NewFilteredListWatchFromClient(client.CoreV1().RESTClient(), "namespaces", "", func(*metav1.ListOptions) {})
```

Reply:
Should work

Reply:
@howardjohn Try NewFilteredXXInformer

Reply:
It does not work either. What is so bad about MultiNamespaceListerWatcher?

Reply:
This isn't a huge deal. It would be nice to have but shouldn't hold up the PR. It can easily be cleaned up later if necessary.